imagersite/imagersite/static/_modules.scss

```diff
@@ -0,0 +1,137 @@
+
+body {
+    background-color: #2e3035;
+    color: white;
+}
+
+.clearfix {
+    clear: both;
+}
+
+h1, h2, h3, h4, a {
+    font-family: 'Fredoka One', cursive;
+}
+
+p {
+    font-family: 'Open Sans', sans-serif;
+}
+
+header {
+    background-image: url("https://s.imgur.com/desktop-assets/desktop-assets/bg-home-2.f1df058d8494e2f402781cc93a7f4003.png");
+    height: 80px;
+}
+
+header #site-title {
```

Comment:
To follow SASS standars you should have this nested inside the header stylings. ( ex. header {stylings: style; #site-title {stylings: style;}} )